source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -455,6 +456,107 @@ message TransformationAccessChain {
+  // |                                             | %18 = OpBitwiseOr %2 %16 %17                                   |
+  // |                                             |                                                                |
+  // |                                             | %19 = OpBitFieldUExtract %2 %5 %6 %6 ; extracts bit 1 from %5  |
+  // |                                             | %20 = OpBitFieldUExtract %2 %6 %6 %6 ; extracts bit 1 from %6  |
+  // |                                             | %21 = OpBitwiseOr %2 %19 %20                                   |
+  // |                                             |                                                                |
+  // |                                             | %22 = OpBitFieldUExtract %2 %5 %10 %6 ; extracts bit 2 from %5 |
+  // |                                             | %23 = OpBitFieldUExtract %2 %6 %10 %6 ; extracts bit 2 from %6 |
+  // |                                             | %24 = OpBitwiseOr %2 %22 %23                                   |
+  // |                                             |                                                                |
+  // |                                             | %25 = OpBitFieldUExtract %2 %5 %11 %6 ; extracts bit 3 from %5 |
+  // |                                             | %26 = OpBitFieldUExtract %2 %6 %11 %6 ; extracts bit 3 from %6 |
+  // |                                             | %27 = OpBitwiseOr %2 %25 %26                                   |
+  // |                                             |                                                                |
+  // |                                             | %28 = OpBitFieldUExtract %2 %5 %12 %6 ; extracts bit 4 from %5 |
+  // |                                             | %29 = OpBitFieldUExtract %2 %6 %12 %6 ; extracts bit 4 from %6 |
+  // |                                             | %30 = OpBitwiseOr %2 %28 %29                                   |
+  // |                                             |                                                                |
+  // |                                             | %31 = OpBitFieldUExtract %2 %5 %13 %6 ; extracts bit 5 from %5 |
+  // |                                             | %32 = OpBitFieldUExtract %2 %6 %13 %6 ; extracts bit 5 from %6 |
+  // |                                             | %33 = OpBitwiseOr %2 %31 %32                                   |
+  // |                                             |                                                                |
+  // |                                             | %34 = OpBitFieldUExtract %2 %5 %14 %6 ; extracts bit 6 from %5 |
+  // |                                             | %35 = OpBitFieldUExtract %2 %6 %14 %6 ; extracts bit 6 from %6 |
+  // |                                             | %36 = OpBitwiseOr %2 %34 %35                                   |
```

Comment:
The example is great but is a bit long and might be annoying for folks browsing the file looking for particular protobuf messages.

How about you replace all this with:

```
// |                                       |  ...                         |
```

That would make it clear that the theme continues similarly.